restaurant-menu/src/containers/Layout.js

```diff
@@ -0,0 +1,143 @@
+import './Layout.css';
+import logoImg from '../assets/img/logo.jpg';
+import React, {useState, useEffect} from 'react';
+import OrderContext from '../OrderContext';
+import Menu from './Menu/Menu';
+import CustomerInfo from './CustomerInfo/CustomerInfo';
+import Spinner from '../components/Spinner/Spinner';
+import axios from 'axios';
+
+
+const Layout = () => {
+
+    let id;
+    const params = new URLSearchParams(document.location.search.substring(1));
+    id = params.get('id');
+    console.log("id: " + id)
+
+    const [status, setStatus] = useState('loading');
+    const [restaurant, setRestaurant] = useState(null);
+    const [foodCategories, setFoodCategories] = useState(null);
+    const [page, setPage] = useState({page: "menu", info: null});
+    
+    const [order, setOrder] = useState({
+        restaurant: id,
+        city: "",
+        street: "",
+        number: "",
+        apartment: "",
+        customer: {
+            phone_number: "",
+            first_name: "",
+            last_name: "",
+            email: ""
+        },
+        remark: "",
+        address_lat: "",
+        address_lng: "",
+        delivery_distance: "",
+        dishes_in_order: []
+    });
```

Comment:
I think it's not the only place where you create an empty order. Best to have a createOrder function in another file that would return an empty order.